db/migrate/20160708144844_add_vendor_id.rb

```diff
@@ -0,0 +1,11 @@
+class AddVendorId < ActiveRecord::Migration
+  def up
+    add_column :donuts, :vendor_id, :integer, null: false
```

Comment:
Because we're still in the prototype stage of this project and don't have anything persisted in our database yet, this is okay. Just be aware that if you did this on a live site, you would be deleting all your data